.github/workflows/kotlin.yml

```diff
@@ -29,23 +35,35 @@ jobs:
           java-version: 1.8
 
       ## Caching
-      - name: Cache build artifacts
+      - name: Cache gradle dependencies
         uses: actions/cache@v1
         with:
-            path: ~/.gradle/caches
-            # Include the SHA in the hash so this step always adds a cache entry. If we didn't use the SHA, the artifacts
-            # would only get cached once for each build config hash.
-            # Don't use ${{ runner.os }} in the key so we don't re-assemble for UI tests.
-            key: gradle-${{ hashFiles('**/gradle/wrapper/gradle-wrapper.properties') }}-${{ hashFiles('**/*.gradle*') }}-${{ hashFiles('**/buildSrc/**') }}-${{ github.sha }}
-            # The first time a SHA is assembled, we still want to load dependencies from the cache.
-            # Note that none of jobs dependent on this one need restore keys, since they'll always have an exact hit.
-            restore-keys: |
-              gradle-${{ hashFiles('**/gradle/wrapper/gradle-wrapper.properties') }}-${{ hashFiles('**/*.gradle*') }}-${{ hashFiles('**/buildSrc/**') }}-
+          path: ${{ env.GRADLE_CACHE_PATH }}
+          # Include the SHA in the hash so this step always adds a cache entry. If we didn't use the SHA, the artifacts
+          # would only get cached once for each build config hash.
+          # Don't use ${{ runner.os }} in the key so we don't re-assemble for UI tests.
+          key: gradle-${{ hashFiles('**/gradle/wrapper/gradle-wrapper.properties') }}-${{ hashFiles('**/*.gradle*') }}-${{ hashFiles('**/buildSrc/**') }}-${{ github.sha }}
+          # The first time a SHA is assembled, we still want to load dependencies from the cache.
+          # Note that none of jobs dependent on this one need restore keys, since they'll always have an exact hit.
+          restore-keys: |
+            gradle-${{ hashFiles('**/gradle/wrapper/gradle-wrapper.properties') }}-${{ hashFiles('**/*.gradle*') }}-${{ hashFiles('**/buildSrc/**') }}-
```

Comment:
What's in `buildSrc`?

Reply:
Right now, just our dependency definitions.